AllReadyApp/Web-App/AllReady/Controllers/TaskApiController.cs

```diff
@@ -1,23 +1,40 @@
-﻿using System;
+﻿using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Mvc;
+
+using AllReady.Security;
+using AllReady.Models;
+using AllReady.ViewModels;
+
+using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net;
 using System.Security.Claims;
-using AllReady.Models;
-using AllReady.Security;
-using AllReady.ViewModels;
-using Microsoft.AspNet.Mvc;
+using System.Threading.Tasks;
+using AllReady.Areas.Admin.Features.Tasks;
+using AllReady.Extensions;
+using AllReady.Features.Activity;
+using AllReady.Features.Notifications;
+using AllReady.Features.Tasks;
+using MediatR;
+using Microsoft.AspNet.Authorization;
+using TaskStatus = AllReady.Areas.Admin.Features.Tasks.TaskStatus;
 
 namespace AllReady.Controllers
 {
+
     [Route("api/task")]
     [Produces("application/json")]
     public class TaskApiController : Controller
     {
         private readonly IAllReadyDataAccess _allReadyDataAccess;
+        private readonly IMediator _bus;
+
 
-        public TaskApiController(IAllReadyDataAccess allReadyDataAccess)
+        public TaskApiController(IAllReadyDataAccess allReadyDataAccess, IMediator bus)
```

Comment:
Just to be consistent...we started by naming `bus` originally, but have switched over to `mediator`. Could you rename this guy?